roles/common/ansible-ssh-config/templates/ansible.ssh.config.j2

```diff
@@ -0,0 +1,22 @@
+{% for host in groups['all'] %}
```

Comment:
Does this file replace the old `templates/ssh.config.ansible.j2`? if so, please remove it